### PR TITLE
Add jenkins build step script

### PR DESCRIPTION
**Pull request by nghiant2710**

### CODE REVIEW

**Comment by petrosagg** [comment]
Can you create a folder named "automation" and put the script in there?

**Comment by nghiant2710** [comment]
@petrosagg , Updated!

File: automation/jenkins_build.sh [hunk 1]
```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+# Set env var
+LIBRARY=$1
```
Comment (petrosagg) [inline automation/jenkins_build.sh]: What does this env var do?

Comment (nghiant2710) [inline automation/jenkins_build.sh]: this env var sets which images will be built

**Comment by nghiant2710** [comment]
@petrossag ^

**Pull request merged**